### PR TITLE
front: bump osrd-ui to use the new manchette props

### LINKED ISSUE #9506: front: bump osrd-ui to 53

### PR DESCRIPTION
**Pull request by clarani**
closes #9506

### CODE REVIEW

**Comment by emersion** [comment]
Note, these two commits should be squashed together because the app won't build in-between.

File: front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx [hunk 1]
```diff
@@ -52,90 +52,101 @@ const ManchetteWithSpaceTimeChartWrapper = ({
...
+      name: waypoint.extensions?.identifier?.name,
+      secondaryCode: waypoint.extensions?.sncf?.ch,
+    }));
+  }, [waypointsPanelData, operationalPoints]);
 
   const { manchetteProps, spaceTimeChartProps, handleScroll } = useManchettesWithSpaceTimeChart(
-    waypointsPanelData?.filteredWaypoints ?? operationalPoints,
-    spaceTimeChartLayersData.filteredProjectPathTrainResult,
+    manchetteWaypoints,
+    cutProjectedTrains,
```
Comment (SharglutDev) [inline front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx:149]: Here we pass the projected trains with `signalUpdates` but the hook doesn't use them. Typescript doesn't seem to mind that there is an extra property in its wanted type but maybe we should remove it ?

File: front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts [hunk 2]
```diff
@@ -1,24 +1,23 @@
-import type { ProjectPathTrainResult } from 'common/api/osrdEditoastApi';
+import type { TrainSpaceTimeData } from 'applications/operationalStudies/types';
```
Comment (SharglutDev) [inline front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts:1]: Is it ok that stdcm depends on a type from operational studies ? We also have to deal with `signalUpdates` even though we don't need them

**Review by SharglutDev — COMMENTED** [review]
Thank you for the PR. Everything works great. Just left a few comments/suggestions

File: front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx (hunk 1 above)
Comment (clarani) [reply front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx:149]: Yes, I think typescript only checks that the required props are given, it does not mind if extra props are given.
To remove the extra props, we will have to add a map. I think it's not very useful and we can stay like this, if it's good for you

File: front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts (hunk 2 above)
Comment (clarani) [reply front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts:1]: No I think that the type TrainSpaceTimeData should be in module/SpaceTimeChart :/
As it was already there, I did not move it. Do you want me to move it now ? I can also do it in another PR.

Concerning the signalUpdates, I think we should always keep it in the TrainSpaceTimeData type, as we might use it every time we are displaying the trains on the SpaceTimeChart

**Comment by clarani** [comment]
@emersion why do you think we should squash the 2 commits ? I can build after the 1st commit (without the 2nd one) in local

**Comment by emersion** [comment]
Hm, right… The first commit bumps half of osrd-ui and the other bumps the rest. I guess that's fine.

File: front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx (hunk 1 above)
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx:149]: Fine by me

File: front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts (hunk 2 above)
Comment (SharglutDev) [reply front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts:1]: Yep, it can be done in a different PR :)

**Review by SharglutDev — APPROVED** [review]

**Pull request merged**